manuscript/setup/dev.md

```diff
@@ -155,9 +165,15 @@ alias curl="curl --insecure"
 # - cncf-demo-dev
 # Do not use a wildcard for those subdomains since, later on,
 #   we'll add more pointing to a different cluster.
+# Configure these subdomains by going to your registrar and creating
+#   three more DNS records of type 'A', each with the name set to the
+#   subdomain (one record for 'harbor', one for 'notary', one for
+#   'cncf-demo-dev'), and the value of each record set to that same IP
+#   address of your output.
 
 # Skip this step if you chose to use `nip.io` instead of a
 #   "real" domain
+# VIKTOR - why do we have to do a 'helm repo add' command for jetstack but not traefik?
```

Comment:
For some, `--repo` argument works so there is no need to do `helm repo add`. For others it doesn't. There might be something I'm missing... It would be nice to get rid of it though.

Reply:
☆彡(ノ^^)ノ thanks for the clarification